static/scss/partials/premium-promo.scss

```diff
@@ -0,0 +1,63 @@
+.premium-promo-hero-cta {
+    text-align: center;
+
+    @media screen and #{$mq-md} {
+        text-align: start;
+    }
+}
+
+
+.premium-promo-hero-cta button:disabled,
+.perk-cta:disabled {
+    cursor: not-allowed;
+    pointer-events: visible;
+}
+
+.premium-promo-availability-warning {
+    padding: $spacing-md 0;
+}
+
+.perks-headline {
+    @include text-title-sm;
+    text-align: center;
+}
+
+.perks-lead {
+    @include text-body-md;
+    display: block;
+
+    @media screen and #{$mq-md} {
+        margin: 0 auto;
+    }
+    width: 75%;
```

Comment:
nit: move `width` above the media query for easier reading

Reply:
Heh, how did that get there :sweat_smile: 4c63f4ba